### ISSUE TITLE
Lepton Tools

### ISSUE THREAD

**Issue by vincecr0ft** [htautau/hhntup#30]
Ok so I'm missing out some obvious things like smearing and calibration and things but even in the event selection the leplep code requires an electron ID parameter of mediumPlusPlus found in https://svnweb.cern.ch/trac/atlasoff/browser/Reconstruction/egamma/egammaAnalysis/egammaAnalysisUtils/tags/egammaAnalysisUtils-00-04-58/Root/IsEMPlusPlusDefs.cxx

There is a value called mediumPP in the ntuples but I'm wary about using it since it removes most of the events and is ignored in the leplep code.

Will this be included in the xAOD Egama tools? do we need to add it to the D3PD. Are there any other tools that we're missing?

**Comment by qbuat** [comment]
Yes, the electron tools will be shipped in the analysis release for the xAODs. You should add the the egammaAnalysisUtils package to externaltools.

**Comment by vincecr0ft** [comment]
cool cool cool. I will let you know how it goes. 

What about systematics? I'm guessing that these aren't really necessary in the D3PD branch right?

**Comment by qbuat** [comment]
Yeah. Not a priority, I think.
